src/webextension/background/browser-action.js

```diff
@@ -37,6 +37,9 @@ export default async function updateBrowserAction(ds) {
   uninstallListener();
   uninstallPopup();
 
+  let iconpath = ds.locked ? "icons/lb_locked.svg" : "icons/lb_unlocked.svg";
```

Comment:
`s/let/const/` please. I've been trying to be as "const-correct" as possible, though it matters significantly less here than in C++.